app/code/Magento/Quote/README.md

```diff
@@ -1,11 +1,310 @@
-# Overview
-## Purpose of module
+# Magento_Quote module
 
+This module provides customer cart management functionality.
 
-# Deployment
-## System requirements
+## Installation
 
-The Magento_Quote module does not have any specific system requirements.
+The Magento_Quote module is one of the base Magento 2 modules. You cannot disable or uninstall this module.
+
+The following modules depend on this module:
+- `Magento_Checkout`
+- `Magento_ConfigurableProduct`
+- `Magento_GiftMessage`
+- `Magento_GroupedProduct`
+- `Magento_OfflineShipping`
+- `Magento_Paypal`
+- `Magento_PaypalGraphQl`
+- `Magento_QuoteAnalytics`
+- `Magento_Reports`
+- `Magento_SalesRule`
+- `Magento_Weee`
+
+The Magento_Quote module creates the following table in the database:
```

Comment:
Please add some information about the schema/column not being persistent (e.g. not having to be manually removed) when the module gets disabled and setup:upgrade is run.

Also - a typo - "table" should be in the plural form. 

Additionally, please cover all the data patches here and mention if modifications introduced in them can have to be manually removed/reverted after the module uninstallation.

Reply:
I don't think we should mention any schema / data reverting as this module should not be disabled or removed.